apps/ui/src/components/SpaceTreasury.vue

```diff
@@ -10,6 +10,7 @@ import {
 } from '@/helpers/utils';
 import { evmNetworks, getNetwork } from '@/networks';
 import { Contact, Space, SpaceMetadataTreasury, Transaction } from '@/types';
+import ICStake from '~icons/c/stake';
```

Comment:
We don't need to import custom icons in Vue templates. Also we in those cases use following format for icons (as any other icon: `<IC-stake />`).